src/x86-64.S

```diff
@@ -1,7 +1,7 @@
 /*
  * This file contains the core of a bitslice DES implementation for x86-64/SSE2.
  * It is part of John the Ripper password cracker,
- * Copyright (c) 2000-2001,2005,2006,2008,2011,2012,2015 by Solar Designer
+ * Copyright (c) 2000-2001,2005,2006,2008,2011,2012,2015,2019 by Solar Designer
```

Comment:
Maybe there should be a copyright statement for magnum's CPU detection code from 2015,2017. Ditto in x86.S.

Reply:
BTW, the AVX2 and AVX512F detection I added to core was indeed inspired by magnum's and I tried to avoid unnecessary differences, but it is slightly different code because we're detecting fewer SIMD instruction sets in core (no SSSE3, no SSE4*). So I felt it can be copyrighted by either or both of us there. But jumbo's is pure magnum's.